sdk/loadtestservice/load-testing-rest/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@azure-rest/load-testing",
   "sdk-type": "client",
   "author": "Microsoft Corporation",
-  "version": "1.0.0-beta.1",
-  "description": "",
+  "version": "1.0.0",
```

Comment:
Is this version correct?

Reply:
will update to beta version as a pre GA release is planned.